plugins/analysis/kotlin-plugin/src/main/kotlin/arrow/meta/plugins/analysis/phases/analysis/solver/ast/kotlin/descriptors/KotlinModuleDescriptor.kt

```diff
@@ -11,7 +11,7 @@ class KotlinModuleDescriptor(val impl: org.jetbrains.kotlin.descriptors.ModuleDe
 
   override fun impl(): org.jetbrains.kotlin.descriptors.ModuleDescriptor = impl
 
-  override fun getPackage(pck: String): PackageViewDescriptor? =
+  override fun getPackage(pck: String): PackageViewDescriptor =
     impl().getPackage(org.jetbrains.kotlin.name.FqName(pck)).model()
```

Comment:
is this true also for functions declared in the root package with no explicit package?

Reply:
I removed the `?` because Kotlin told me that it was fine.